OpenRA.Mods.Cnc/Traits/MadTank.cs

```diff
@@ -216,14 +148,94 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 
 				self.SetTargetLine(order.Target, Color.Red);
 				self.QueueActivity(new MoveAdjacentTo(self, order.Target, targetLineColor: Color.Red));
```

Comment:
Can we please handle this movement within `DetonationSequence`, like we do for other activities?

Reply:
Done.